skt/kernelbuilder.py

```diff
@@ -27,19 +27,53 @@
 from threading import Timer
 
 
+def verify_target(target):
```

Comment:
The name verify, implies raising an exception or return True or False to me. It's something more like translate_target and judging for the implementation a dict satisfies the necessity very well instead of matching word by word.